ui/src/components/view/SearchView.vue

```diff
@@ -53,8 +53,7 @@
                   v-for="(field, index) in fields"
                   :key="index"
                   :label="field.name==='keyword' ?
-                    ('listAnnotations' in $store.getters.apis ? $t('label.annotation') : $t('label.name')) :
-                    (field.name==='entitytype' ? $t('label.entity.type') : $t('label.' + field.name))">
+                    ('listAnnotations' in $store.getters.apis ? $t('label.annotation') : $t('label.name')) : $t('label.' + field.name)">
```

Comment:
@davidjumani is your syntax correct: `condition ? label1 : label2 : label 3` ?

Reply:
```
condition-1 ? 
    (condition-2 ? label1 : label2) : label3
```